index.js

```diff
@@ -0,0 +1,354 @@
+			con.query("select * from events;", function(err, events_data) {
+				res.render("dashboard_facility_card", {
+					res: req.session.user,
+					booking_data,
+					events_data
+				});
+			});
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/dashboard_facility_table", (req, res) => {
+	if (req.session.user) {
+		con.query("select * from booking;", function(err, booking_data) {
+			con.query("select * from events;", function(err, events_data) {
+				res.render("dashboard_facility_table", {
+					res: req.session.user,
+					booking_data,
+					events_data
+				});
+			});
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/request", (req, res) => {
+	if (req.session.user) {
+		res.render("request");
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/logout", (req, res) => {
+	res.clearCookie("session", {
+		path: "/"
+	});
+	res.redirect("/login");
+});
+app.post("/onLogin", (req, res) => {
+	var username = req.body.username;
+	var password = req.body.password;
+	con.query(
+		"select * from users where username='" + username + "';",
+		function(err, rows) {
+			if (!err) {
+				if (rows.length > 0) {
+					if (
+						password ===
+						AES.decrypt(rows[0].password, `${process.env.AES_key}`)
+					) {
+						req.session.user = rows[0];
+						res.redirect("/dashboard");
+					} else {
+						console.log("Invalid password");
+						res.status(401).render("error", {
+							error_message: "Authentication error!"
+						});
+					}
+				} else {
+					console.log("Username not found!");
+					res.status(401).render("error", {
+						error_message: "Authentication error!"
+					});
+				}
+			} else {
+				console.log("Username not found!");
+				res.status(401).render("error", {
+					error_message: "Authentication error!"
+				});
+			}
+		}
+	);
+});
+app.post("/checkAvailability", (req, res) => {
+	if (req.session.user) {
+		con.query(
+			`SELECT slot_hall FROM booking WHERE event_id IN (SELECT id FROM events WHERE date_wanted = '${req.body.date_wanted}')`,
+			(err, result) => {
+				if (!err) {
+					res.send(result);
+				} else {
+					console.log(err);
+					res.status(500).render("error", {
+						error_message: "Availability checking failed!"
+					});
+				}
+			}
+		);
+	} else {
+		res.redirect("/login");
+	}
+});
+app.post("/makeRequest", (req, res) => {
+	if (req.session.user) {
+		var user_id = req.session.user.id;
+		var date_wanted = req.body.date_wanted;
+		var slot_hall = "";
+		for (var i = 1; i <= 7; i++)
+			for (var j = 1; j <= 9; j++) {
+				if (req.body[`cell${j}${i}`]) {
+					slot_hall += j.toString() + i.toString();
+				}
+			}
+		var club = req.body.club_name;
+		var details = req.body.desc;
+		var title = req.body.event_name;
+		var requirements = req.body.requirements;
+		var event_obj = {
+			title: title,
+			club: club,
+			requirements: requirements,
+			details: details,
+			date_wanted: date_wanted,
+			user_id: user_id
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'user_id' is not in camel case. (camelcase)](https://app.codacy.com/app/AravindVNair99/Hall-Management-System/pullRequest?prid=4592231)